css: Move alert.scss to common/alert.scss.

wryonik opened the pull request:
This PR is a follow up of #14411 .It moves "static/styles/alert.scss" to "static/styles/common/alert.scss" so that it can be used in portico pages as well.

wryonik committed:
css: Move alert.scss to common/alert.scss.

This commit moves "static/styles/alert.scss" to
"static/styles/common/alert.scss" so that it can be used in portico
pages as well.

hackerkid commented:
I think you should now include `/styles/common/alerts.scss"` in `common.js` since that file is required by both `app.js` and `portico.js`. 

That means you would have to change various portico files that has `alert` and other classes in `alerts.scss` to fit the new styling changes. Changes like showing the alert when required for example since the behavior of alert class used currently in portico is to be shown while that of `alert.scss` is to be hidden.

I would wait for @timabbott to confirm this is the direction we want to take before doing further work on this.

timabbott commented:
I think it's a useful direction to merge these definitions of `alert`, but I also agree with Vishnu that we need to do do those changes, likely delete the existing `alert` CSS on portico pages, and then actually test every single use of the alert classes on the portico pages.

wryonik committed:
portico: Rename existing alert classes on portico.

The "alert.scss" was recently transferred to
"static/styles/common/alert.scss" so that common CSS can be shared
between main app pages and portico pages. Since it may also affect the
alert in portico pages, this commit renames the alert classes used in
portico pages so that the common CSS may not conflict with each other.

shubhamgupta2956 commented:
@hackerkid I had updated the PR by renaming the alert classes in portico pages to `portico-alert*` like naming (I could not think of a better name). I had tested all the use of new alert classes in the portico to the best of my knowledge. Please review it.
The circleci test is failing due to an error that is reported in automated testing.

timabbott commented:
@majordwarf can you help review this?

timabbott commented:
@SiddharthVarshney can you help review this?
